eng/scripts/Analyze-Code.ps1

```diff
@@ -24,6 +25,10 @@ if ($CheckWasm) {
   Invoke-LoggedCommand "rustup target add --toolchain $Toolchain wasm32-unknown-unknown"
 }
 
+if ($Deny) {
+  Invoke-LoggedCommand "cargo +$Toolchain install cargo-deny --locked"
```

Comment:
@hallipr we might actually want to move this out of the script - the install, specifically - so we can take advantage of build caching. Or maybe we can still do that if we collect the right directories in Azure Pipelines. For GitHub Actions, I do this: <https://github.com/heaths/template-rustlang/blob/2b23770467f06c59977edfbeea84e96b144a702e/.github/workflows/ci.yml#L37-L45>

Reply:
Shouldn't we still be able to use build caching.  It may be missing cargo-deny, but most other packages and the target folder will be cached

Reply:
Should, yes. It's basically just a `cargo build` of an external crate. All dependencies end up in the same place. But is that part of the "analyze" step already?

Whatever the case, this shouldn't block this PR so I'm resolving the comment.